### ISSUE TITLE
Feature: push devlogs

### ISSUE THREAD

**Issue by Guard13007** [itchio/butler#148]
As mentioned in itchio/itch.io#677, it would be nice to be able to push a devlog from butler.

I'm imagining something similar to how git does commit messages, though I am curious if @charliesgames has a different idea.

**Comment by charliesgames** [comment]
That was pretty much what i was thinking.

**Comment by colinbellino** [comment]
I agree this would be a great addition to butler, but is this even possible with the itch.io API? (i'm looking for some doc on the matter)

**Comment by fasterthanlime** [comment]
No, there is no API support for doing this at the moment.

**Comment by colinbellino** [comment]
Alright, fair enough. Is this something the team would like to add at some point or is this totally out of scope ?

Also, sorry for reopening an old issue, didn't see it was from 2018.
